crates/re_viewer/src/ui/welcome_screen/welcome_page.rs

```diff
@@ -63,19 +66,30 @@ fn onboarding_content_ui(
             image: &re_ui::icons::WELCOME_SCREEN_LIVE_DATA,
             add_buttons: Box::new(|ui: &mut egui::Ui| {
                 //TODO(#3870): enable with C++ guides are completed
-                #[allow(clippy::collapsible_if)]
+                #[allow(clippy::collapsible_if, clippy::if_same_then_else)]
                 if false {
                     if large_text_button(ui, "C++").clicked() {
+                        let (markdown, code) = if accepts_connections {
+                            (
+                                include_str!("../../../data/quick_start_guides/cpp_connect.md"),
+                                include_str!(
+                                    "../../../data/quick_start_guides/quick_start_connect.cpp"
+                                ),
+                            )
+                        } else {
+                            (
+                                include_str!("../../../data/quick_start_guides/cpp_spawn.md"),
+                                include_str!(
+                                    "../../../data/quick_start_guides/quick_start_spawn.cpp"
```

Comment:
I think this code could become a bit more readable you moved all the `include_str`:s to the top of the file instead

Reply:
Good call!